core/templates/components/forms/custom-forms-directives/edit-thumbnail-modal.component.ts

```diff
@@ -89,6 +91,12 @@ export class EditThumbnailModalComponent implements OnInit {
     return this.uploadedImageMimeType === 'image/svg+xml';
   }
 
+  isValidFilename(file: File): boolean {
+    const VALID_THUMBNAIL_FILENAME_REGEX = new RegExp(
+      constants.VALID_THUMBNAIL_FILENAME_REGEX);
+    return VALID_THUMBNAIL_FILENAME_REGEX.test(file.name);
```

Comment:
Does this work instead?
```suggestion
    return file.name.match(constants.VALID_THUMBNAIL_FILENAME_REGEX);
```

Reply:
Yes should work, i'll check and let you know

Reply:
Does not work as .match() returns the array of matched strings rather than boolean 
Due to which we have to change all test regarding to this function